plugins/javadoc/src/main/kotlin/org/jetbrains/dokka/javadoc/pages/JavadocPageNodes.kt

```diff
@@ -262,9 +272,60 @@ class AllClassesPage(val classes: List<JavadocClasslikePageNode>) : JavadocPageN
 
 }
 
+class DeprecatedPage(
+    val elements: IdentityHashMap<DeprecatedPageSection, MutableSet<DeprecatedNode>>,
```

Comment:
I really don't like mutability in page model. Why cannot it be expressed immutably?